rllib/tests/test_supported_spaces.py

```diff
@@ -69,6 +69,11 @@ def _do_check(alg, config, a_name, o_name):
 
         try:
             a = get_trainer_class(alg)(config=config, env=RandomEnv)
+        except ray.exceptions.RayActorError as e:
```

Comment:
NIT: is there a way to check for a tighter error than just RayActorError? AFAICT this could mean a host of issues with the actor, not just issues with the space of the environment.

Reply:
We could maybe check the actual error message. You are right, we would miss any other error thrown on any of the workers (even if it's not a space exception) and interpret it as a "harmless" space-not-supported exception, which it may not be.

Reply:
Actually, no, we do this already here:

```
            if isinstance(e.args[2], UnsupportedSpaceException):
                stat = "unsupported"
            else:
                raise
```

So only an underlying `UnsupportedSpaceException` makes this test know, it's harmless. All other underlying error types are re-raised.